cmd/wallet-sdk-gomobile/didresolver/resolver.go

```diff
@@ -8,6 +8,8 @@ SPDX-License-Identifier: Apache-2.0
 package didresolver
 
 import (
+	// helps gomobile bind api.DIDResolver interface to Resolver implementation in ios-bindings.
```

Comment:
Just curious what issue you ran into here? (I don't see any problem with doing this, just want to know for my own knowledge)

Reply:
without this import, it generates Resolve that not implements the DIDResolve interface. 
Previously it was :
`@interface DidresolverResolver : NSObject <goSeqRefInterface> {
}`

With import it becomes

`@interface DidresolverResolver : NSObject <goSeqRefInterface, ApiDIDResolver> {
}` 

so now DidresolverResolver can be cast to ApiDIDResolver

Reply:
Just to easily understand this is wired Objective-C syntax, analog on Java will look like that.

`class DidresolverResolver implements ApiDIDResolver> {  }`